home/templates/home/criteria.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+
+<H1>Judging Criteria for Hackaton</H1>
+<br>
+<ul>
+    <li>Placeholder text</li>
+    <li>Content</li>
+    <li>Structure and Navigation</li>
+    <li>Visual Design</li>
+    <li>Functionality</li>
+    <li>Interactivity</li>
+    <li>Innovation</li>
+    <li>Overall Experience</li>
+</ul>
+<br>
+<h2><b>Content</b></h2>
```

Comment:
Could you please replace the `<b>` tags with `<strong>` . Alternatively, you could create a custom css class called `bold` (or similar) and add the class to the `<h2>`.